src/Servers/Kestrel/test/Interop.FunctionalTests/Http3/Http3RequestTests.cs

```diff
@@ -1606,6 +1606,63 @@ public async Task GET_ClientDisconnected_ConnectionAbortRaised()
         }
     }
 
+    [ConditionalFact]
+    [MsQuicSupported]
+    public async Task GET_ClientIdlesOut_ConnectionAbortRaised()
```

Comment:
I'm assuming this used to trigger the assert described in https://github.com/dotnet/aspnetcore/issues/55196 and that's why it's a useful regression test.  As a standalone test, however, it might be useful to check the error code as well.  It could even set `DefaultCloseErrorCode` and/or `DefaultCloseErrorCode` to confirm that the configuration is respected.

Reply:
When I try to run this locally, I get 
```
| System.ObjectDisposedException: Cannot access a disposed object.
| Object name: 'System.Net.Quic.QuicListener'.
|    at System.Net.Quic.QuicListener.DisposeAsync()
```

Reply:
It also fails without the product change, so it's not related to the PR.

Reply:
I've pushed a change that I think will do this but, as noted above, I can't fully validate locally.  Let's see what CI says.